settings/DevHome.Settings/Strings/en-us/Resources.resw

```diff
@@ -58,10 +58,6 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="NavigationPane.Content" xml:space="preserve">

```

Comment:
AFAIK we're hiding the navigation header for all pages in favor of a custom layout/design per page (?). So we can probably remove all the `NavigationPane`. Not necessarily in this PR. Resolving.